core/src/main/scala/chisel3/internal/SourceInfoMacro.scala

```diff
@@ -0,0 +1,23 @@
+package chisel3.internal.sourceinfo
```

Comment:
```suggestion
// SPDX-License-Identifier: Apache-2.0

package chisel3.internal.sourceinfo
```

This needs a license specifier